wrapper/src/main/java/io/opentelemetry/instrumentation/awslambda/v1_0/TracingSqsEventWrapper.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright Splunk Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.instrumentation.awslambda.v1_0;
+
+import com.amazonaws.services.lambda.runtime.Context;
+import com.amazonaws.services.lambda.runtime.events.SQSEvent;
+import io.opentelemetry.sdk.OpenTelemetrySdk;
+import io.opentelemetry.sdk.autoconfigure.OpenTelemetrySdkAutoConfiguration;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+
+/** Class will be donated to OpenTelemetry Java Instrumentation project. */
+public class TracingSqsEventWrapper extends TracingSqsEventHandler {
+
+  private final WrappedLambda wrappedLambda;
+
+  public TracingSqsEventWrapper() {
+    this(OpenTelemetrySdkAutoConfiguration.initialize(), WrappedLambda.fromConfiguration());
+  }
+
+  // Visible for testing
```

Comment:
What testing? 😄

Reply:
Future one :P

Reply:
If this isn't actually being used for testing, let's keep it private and expose it when we have tests that need it.

Reply:
The whole class will move to OTEL - it's convention there to have this constructor (and use it for testing), so I put it already in place. The reason it's here now is just to speed up delivery of this feature.